### ISSUE TITLE
fix, document, or remove storefuns

### ISSUE THREAD

**Issue by mahrud** [Macaulay2/M2#1980]
This is an undocumented feature:
https://github.com/Macaulay2/M2/blob/f0ce581750dfd94336883695d3552671410f32f5/M2/Macaulay2/m2/methods.m2#L640-L649
Used probably only here for manually implanting a resolution on an object:
https://github.com/Macaulay2/M2/blob/f0ce581750dfd94336883695d3552671410f32f5/M2/Macaulay2/m2/res.m2#L208-L209

I suppose this is somewhat similar to the `forceGB` function, but much more trusting of the user and hacky:
```m2
i1 : R = kk[x];

i2 : I = ideal x;

o2 : Ideal of R

i3 : res I = 5

o3 = 5

i4 : res I

      1      1
o4 = R  <-- R  <-- 0
                    
     0      1      2

o4 : ChainComplex

i5 : M = module R;

i6 : res M = 5

o6 = 5

i7 : res M

o7 = 5
```

It should either be removed, or fixed and documented. Once #1918 is merged, it is also possible to do something like this by manually storing a value in cache, similar to this:
https://github.com/Macaulay2/M2/blob/f0ce581750dfd94336883695d3552671410f32f5/M2/Macaulay2/packages/PrimaryDecomposition.m2#L165-L169
Though I'm not a fan of this syntax, and would be open to a general suggestion.

I think it would be good to support such functionality beyong `gb` and `res`, for instance `syz`, `quotient`, `saturate`, `intersect`, etc.

**Comment by DanGrayson** [comment]
Actually, it's documented at https://faculty.math.illinois.edu/Macaulay2/doc/Macaulay2-1.17/share/doc/Macaulay2/Macaulay2Doc/html/__eq.html , and I probably use it in several places.  I quite like it.

**Comment by mahrud** [comment]
Wait, I don't see it. Where does it talk about storefuns or res?

**Comment by DanGrayson** [comment]
Ah, I was mixing that up with assignment methods for binary operators.  It seems to be that `storefuns` and the single use of it in `res` is redundant and might as well be eliminated, as you suggest.

It would be nice to have assignment methods for methods.
